src/syscheckd/registry/registry.c

```diff
@@ -509,206 +534,154 @@ void fim_registry_free_entry(fim_entry *entry) {
     }
 }
 
-/**
- * @brief Process and trigger delete events for a given registry value.
- *
- * @param fim_sql An object holding all information corresponding to the FIM DB.
- * @param data A fim_entry object holding the deleted value information retrieved from the FIM DB.
- * @param mutex A mutex to be locked before operating on the registry tables from the FIM DB.
- * @param _alert A pointer to an integer specifying if an alert should be generated.
- * @param _ev_mode A value specifying if the event has been triggered in scheduled, realtime or whodata mode.
- * @param _w_evt A whodata object holding information corresponding to the event.
- */
-void fim_registry_process_value_delete_event(fdb_t *fim_sql,
-                                             fim_entry *data,
-                                             __attribute__((unused)) pthread_mutex_t *mutex,
-                                             void *_alert,
-                                             void *_ev_mode,
-                                             __attribute__((unused)) void *_w_evt) {
-    int alert = *(int *)_alert;
-    fim_event_mode event_mode = *(fim_event_mode *)_ev_mode;
-    registry_t *configuration;
+cJSON* fim_dbsync_registry_value_json_event(const cJSON* dbsync_event,
+                                            const registry_t *configuration,
+                                            fim_event_mode mode,
+                                            const event_data_t* evt_data,
+                                            __attribute__((unused)) whodata_evt *w_evt,
+                                            const char* diff,
+                                            const int data_type)
+{
 
-    configuration = fim_registry_configuration(data->registry_entry.key->path, data->registry_entry.key->arch);
-    if (configuration == NULL) {
-        return;
-    }
+    //cJSON *changed_attributes = NULL;
 
-    if (alert) {
-        cJSON *json_event = fim_registry_event(data, NULL, configuration, event_mode, FIM_DELETE, NULL, NULL);
+    /*if (old_data != NULL && old_data->registry_entry.value != NULL) {
+        changed_attributes = fim_registry_compare_value_attrs(new_data->registry_entry.value,
+                                                              old_data->registry_entry.value, configuration);
 
-        if (json_event) {
-            send_syscheck_msg(json_event);
-            cJSON_Delete(json_event);
+        if (cJSON_GetArraySize(changed_attributes) == 0) {
+            cJSON_Delete(changed_attributes);
+            return NULL;
         }
-    }
-    /* DEPRECATED CODE
-    fim_db_remove_registry_value_data(fim_sql, data->registry_entry.value);
-    */
-    if (configuration->opts & CHECK_SEECHANGES) {
-        fim_diff_process_delete_value(data->registry_entry.key->path, data->registry_entry.value->name,
-                                      data->registry_entry.key->arch);
-    }
-}
+    }*/
```

Comment:
If this code is useless, you can delete it

Reply:
Done in [commit](https://github.com/wazuh/wazuh/pull/11239/commits/11a6215594b89425c4469067a26e23d1a1a7f5df)